lightning/backend/local.go

```diff
@@ -1608,18 +1692,23 @@ func (w *LocalWriter) writeRowsLoop() {
 		totalSize += batchSize
 		log.L().Info("write data by sort index", zap.Int64("bytes", totalSize))
 	}
+	w.local.lockForImport(importMutexStateNoLock)
```

Comment:
```suggestion
	w.local.lockForImport(importMutexStateIngest)
```
Since we define enum for each lock action, should avoid using the no-lock when actually do lock

Reply:
But which value shall we set for `unlock`?

Reply:
If we set it to `importMutexStateIngest ` when calling `unlock`, it seems too strange....

Reply:
Unlock should always reset the value to `importMutexStateNoLock`, if most parameter for `lockForImport` is `importMutexStateNoLock`, this enum makes little sense, just replace it with `lock` (only lock but not set the enum), and `lockForImport` (lock and set importMutexStateImporting) is clearer.